esp-box-3/include/bsp/esp-box-3.h

```diff
@@ -0,0 +1,491 @@
+ *
+ * There are multiple devices connected to I2C peripheral:
+ *  - Codec ES8311 (configuration only)
+ *  - ADC ES7210 (configuration only)
+ *  - Encryption chip ATECC608A (NOT populated on most boards)
+ *  - LCD Touch controller
+ *  - Inertial Measurement Unit ICM-42607-P
+ *
+ * After initialization of I2C, use BSP_I2C_NUM macro when creating I2C devices drivers ie.:
+ * \code{.c}
+ * es8311_handle_t es8311_dev = es8311_create(BSP_I2C_NUM, ES8311_ADDRRES_0);
+ * \endcode
+ **************************************************************************************************/
+#define BSP_I2C_NUM     CONFIG_BSP_I2C_NUM
+
+/**
+ * @brief Init I2C driver
+ *
+ * @return
+ *      - ESP_OK                On success
+ *      - ESP_ERR_INVALID_ARG   I2C parameter error
+ *      - ESP_FAIL              I2C driver installation error
+ *
+ */
+esp_err_t bsp_i2c_init(void);
+
+/**
+ * @brief Deinit I2C driver and free its resources
+ *
+ * @return
+ *      - ESP_OK                On success
+ *      - ESP_ERR_INVALID_ARG   I2C parameter error
+ *
+ */
+esp_err_t bsp_i2c_deinit(void);
+
+/**************************************************************************************************
+ *
+ * SPIFFS
+ *
+ * After mounting the SPIFFS, it can be accessed with stdio functions ie.:
+ * \code{.c}
+ * FILE* f = fopen(BSP_SPIFFS_MOUNT_POINT"/hello.txt", "w");
+ * fprintf(f, "Hello World!\n");
+ * fclose(f);
+ * \endcode
+ **************************************************************************************************/
+#define BSP_SPIFFS_MOUNT_POINT      CONFIG_BSP_SPIFFS_MOUNT_POINT
+
+/**
+ * @brief Mount SPIFFS to virtual file system
+ *
+ * @return
+ *      - ESP_OK on success
+ *      - ESP_ERR_INVALID_STATE if esp_vfs_spiffs_register was already called
+ *      - ESP_ERR_NO_MEM if memory can not be allocated
+ *      - ESP_FAIL if partition can not be mounted
+ *      - other error codes
+ */
+esp_err_t bsp_spiffs_mount(void);
+
+/**
+ * @brief Unmount SPIFFS from virtual file system
+ *
+ * @return
+ *      - ESP_OK on success
+ *      - ESP_ERR_NOT_FOUND if the partition table does not contain SPIFFS partition with given label
+ *      - ESP_ERR_INVALID_STATE if esp_vfs_spiffs_unregister was already called
+ *      - ESP_ERR_NO_MEM if memory can not be allocated
+ *      - ESP_FAIL if partition can not be mounted
+ *      - other error codes
+ */
+esp_err_t bsp_spiffs_unmount(void);
+
+/**************************************************************************************************
+ *
+ * SD card
+ *
+ * After mounting the SD card, it can be accessed with stdio functions ie.:
+ * \code{.c}
+ * FILE* f = fopen(BSP_MOUNT_POINT"/hello.txt", "w");
+ * fprintf(f, "Hello %s!\n", bsp_sdcard->cid.name);
+ * fclose(f);
+ * \endcode
+ *
+ * @attention IO2 is also routed to RGB LED and push button
+ **************************************************************************************************/
+#define BSP_MOUNT_POINT      CONFIG_BSP_SD_MOUNT_POINT
```

Comment:
```suggestion
#define BSP_SD_MOUNT_POINT      CONFIG_BSP_SD_MOUNT_POINT
```
Sorry I overlooked it. I see, that it is bad in Korvo-2.